docs/diagrams/FindEventSequenceDiagram.puml

```diff
@@ -0,0 +1,69 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":FindEventCommandParser" as FindEventCommandParser LOGIC_COLOR
+participant "d:FindEventCommand" as FindEventCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":EventNameContainsKeywordsPredicate" as EventNameContainsKeywordsPredicate MODEL_COLOR
+end box
+
+[-> LogicManager : execute("findevent concert")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("findevent concert")
+activate AddressBookParser
+
+create FindEventCommandParser
+AddressBookParser -> FindEventCommandParser
+activate FindEventCommandParser
+
+FindEventCommandParser --> AddressBookParser
+deactivate FindEventCommandParser
+
+AddressBookParser -> FindEventCommandParser : parse("concert")
+activate FindEventCommandParser
+
+create EventNameContainsKeywordsPredicate
+FindEventCommandParser -> EventNameContainsKeywordsPredicate : EventNameContainsKeywordsPredicate("concert")
+activate EventNameContainsKeywordsPredicate
+
+EventNameContainsKeywordsPredicate--> FindEventCommandParser
+deactivate EventNameContainsKeywordsPredicate
+
+create FindEventCommand
+FindEventCommandParser -> FindEventCommand
+activate FindEventCommand
+
+FindEventCommand --> FindEventCommandParser : f
+deactivate FindEventCommand
```

Comment:
Do you think we need to destroy this too (along with its Parser which is destroyed already)?

Reply:
This I'm honestly not very sure, I would think to leave it as it is since the destroy symbol is kind of optional anyways